app/views/home/_header.html.erb

```diff
@@ -1,30 +1,31 @@
-<nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+<nav class="navbar navbar-light bg-secondary fixed-top">
   <div class="container-fluid">
-    <%= link_to "Simform", root_path, class:"navbar-brand" %>
     
-    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <%= link_to "GADA Electronics", root_path, class:"navbar-brand" %>
+    <button class="navbar-toggler" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvasNavbar" aria-controls="offcanvasNavbar">
       <span class="navbar-toggler-icon"></span>
     </button>
-    <div class="collapse navbar-collapse" id="navbarSupportedContent">
-      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-        
-        <li class="nav-item">
-          <%= link_to "About us", home_about_path, class:"nav-link" %>  
-        </li>
-
-        <li class="nav-item">
-          <%= link_to "Team", employees_path, class:"nav-link" %>     
-        </li>
+    <div class="offcanvas offcanvas-end" tabindex="-1" id="offcanvasNavbar" aria-labelledby="offcanvasNavbarLabel">
+      <div class="offcanvas-header">
+        <h5 class="offcanvas-title" id="offcanvasNavbarLabel">GADA Electronics</h5>
+        <button type="button" class="btn-close text-reset" data-bs-dismiss="offcanvas" aria-label="Close"></button>
+      </div>
+      <div class="offcanvas-body">
+        <ul class="navbar-nav justify-content-end flex-grow-1 pe-3">
+          <li class="nav-item">
+            <%= link_to "Home", root_path, class:"navbar-brand" %>
+          </li>
+          <li class="nav-item">
+            <%= link_to "About us", home_about_path, class:"navbar-brand" %>    
+          </li>
 
-        <li class="nav-item">
-          <%= link_to "Add new Member", new_employee_path, class:"nav-link" %>  
-        </li>
+          <li class="nav-item">
+            <%= link_to "Manage Stocks", products_path, class:"navbar-brand" %>
+          </li>
+        
+        </ul>
         
-      </ul>
-      <form class="d-flex">
-        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
-        <button class="btn btn-outline-success" type="submit">Search</button>
-      </form>
+      </div>
     </div>
   </div>
 </nav>
```

Comment:
add proper indentation and leave line at end of file

Reply:
done